CompanyEmployees.Core.Domain/Repositories/IAttendanceRepository.cs

```diff
@@ -0,0 +1,13 @@
+using CompanyEmployees.Core.Domain.Entities;
+
+namespace CompanyEmployees.Core.Domain.Repositories;
+
+public interface IAttendanceRepository
+{
```

Comment:
Consider making the return type of `GetAttendanceAsync` nullable (`Attendance?`) to indicate it may return null if the attendance ID doesn't match any entity. This aligns with modern C# nullability best practices and avoids ambiguity for consumers of the API.

**Suggested change**
```suggestion
Task<Attendance?> GetAttendanceAsync(Guid attendanceId, bool trackChanges, CancellationToken ct = default);
```